src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap

```diff
@@ -330,7 +330,7 @@ exports[`StoryPromo Container should render audio correctly for amp 1`] = `
           <span
             className="c7"
           >
-            Ọdịyo
+            Audio
```

Comment:
Can you make the test run igbo instead of news please? This test implictly tests the translation is passed from config and isnt hardcoded, having it english doesn't do that as much